kube-fluentd/gcp/Gemfile

```diff
@@ -0,0 +1,7 @@
+source "https://rubygems.org"
+
+gem 'fluentd', '~>0.12.31'
```

Comment:
`fluent-plugin-google-cloud` depends on fluentd version 0.14. Are you intentionally downgrading it? Won't installing the other gem pull in both?